core/schema/cache.go

```diff
@@ -62,12 +62,12 @@ func (s *cacheSchema) cacheVolume(ctx context.Context, parent dagql.Instance[*co
 			return inst, err
 		}
 
-		namespaceKey = name + symbolic
+		namespaceKey = "mod(" + name + symbolic + ")"
```

Comment:
Note: this is changed to include `mod` so that no module can be cleverly named `internal` to collide with the internal cache.